datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxClusterDetailResponse.java

```diff
@@ -33,7 +33,8 @@ public void setStackV4Response(StackV4Response stackV4Response) {
 
     @Override
     public String getTagValue(String key) {
-        return Optional.ofNullable(stackV4Response.getTags())
+        return Optional.ofNullable(stackV4Response)
+                .map(stack -> stack.getTags())
```

Comment:
looks like, `getTags()` can be null

Reply:
yeah, that's why mapping it. if it is null then the lambda evaluates it as an empty result hence the - expected - null shall return